### DISCUSSION TITLE
How to disable tracking on a <mux-video> element

### DISCUSSION THREAD

**Discussion by bennlich** [muxinc/elements#614] [Q&A] [answered]
Hiya. I notice that my mux-video elements make a lot of HTTP requests to inferred.litix.io. I assume this has to do with the MUX Data playback tracking. Is there a way to disable these requests? I am not interested in tracking playback statistics.

**Comment by luwes** [comment]
Thanks for the question! Currently there is no way to disable Mux data on this element. 
it's a good suggestion though, I'll bring this up with the team!

**Comment by arshamg** [comment]
@luwes any update on this? This is a high priority feature for our team. 

We use a custom beacon domain but we don't have this active when developing locally. This means we see a stream of errors from Mux in the console and it's a DX pain.

**Reply by luwes** [reply]
sorry, it's not yet implemented but we want to add support for this.

**Comment by luwes** [comment, marked as answer]
This feature has been released in [@mux/mux-video@0.18.0](https://github.com/muxinc/elements/releases/tag/%40mux%2Fmux-video%400.18.0)
See an example at https://codesandbox.io/p/sandbox/mux-video-disable-tracking-krwmxl?file=%2Findex.html%3A17%2C23

**Comment by ghost** [comment]
I have it blocked on my Firewall. I cannot stress how important it is to have a firewall that YOU manage. I love my Firewalla. You would not believe how much garbage I have kicked off my network. But, you do whatever is best for your situation.
